bindings/binding.cpp.tmpl

```diff
@@ -0,0 +1,16 @@
+// This file was autogenerated by the Chimera build.
```

Comment:
I intentionally did this because I was a little worried that the auto-generated variables might somehow contain something that could be scoped incorrectly inside the `chimera::binding` namespace.

Probably unlikely, but it doesn't really get much shorted if you add the `namespace {` blocks in here anyway.  In fact, I'm not sure it saved _any_ space when I tried it.